ci-operator/step-registry/ipi/conf/nutanix/extended/ipi-conf-nutanix-extended-commands.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+CONFIG="${SHARED_DIR}/install-config.yaml"
+
+echo "$(date -u --rfc-3339=seconds) - Adding platform extended data to install-config.yaml"
+
+sed -i "/^  nutanix:/a\    defaultMachinePlatform:\n      bootType: ${VMS_BOOTTYPE}\n      categories:\n      - key: ${CATEGORIES_KEY}\n        value: ${CATEGORIES_VALUE}\n      project:\n        name: ${VMS_PROJECT}\n        type: name" "${CONFIG}"
```

Comment:
suggest to create a yaml patch file, then use `yq` command to patch the install-config.yaml

Reply:
right.